index.html

```diff
@@ -217,8 +217,6 @@ <h1 id="ovningar">Övningar</h1>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
     <script src="vendor/highlight.pack.js"></script>
     <script src="vendor/bootstrap.min.js"></script>
-    <script src="vendor/smooth-scroll.js"></script>
```

Comment:
> Yeah, it exists, but I mean, marquee and blink was popular in the 90s.

Hardly comparable to smooth scrolling anchor links, buddy.

- The Ruby server is a one time thing for getting a GitHub API token, but I agree it could've been written in Node.js too, but it was very simple to just drop it in during development.
- The exercise files are a bit spread out, it was made after I wrote the initial stuff.
- The gulpfile is a common workflow for building Sass etc.

All in all, a quite common Node stack. `npm install` and off you go.